tokio/tests/io_panic.rs

```diff
@@ -70,7 +71,6 @@ fn read_buf_initialize_unfilled_to_panic_caller() -> Result<(), Box<dyn Error>>
     Ok(())
 }
 
-#[cfg(panic = "unwind")]
```

Comment:
Could you remove #[cfg(panic = "unwind")] of read_buf_initialize_unfilled_to_panic_caller (line 57) as well?

Reply:
Done.